tests/e2e/performance/test_fio_benchmark.py

```diff
@@ -327,23 +355,38 @@ def run(self):
         # Getting the UUID from inside the benchmark pod
         self.uuid = self.ripsaw.get_uuid(self.client_pod)
         # Setting back the original elastic-search information
-        if self.backup_es:
+        if hasattr(self, "backup_es"):
             self.crd_data["spec"]["elasticsearch"] = self.backup_es
         if self.dev_mode:
-            timeout = 600
             sleeptime = 30
         else:
-            timeout = 18000
             sleeptime = 300
 
-        self.wait_for_wl_to_finish(timeout=timeout, sleep=sleeptime)
+        self.wait_for_wl_to_finish(sleep=sleeptime)
 
         try:
             if "Fio failed to execute" not in self.test_logs:
                 log.info("FIO has completed successfully")
         except IOError:
             log.warning("FIO failed to complete")
 
+    def teardown(self):
+        """
+        The teardown of the test environment in the end.
+
+        """
+        log.info("cleanup the environment")
+        if hasattr(self, "ripsaw"):
+            self.ripsaw.cleanup()
+        if isinstance(self.es, ElasticSearch):
+            self.es.cleanup()
+
+        sleep_time = 5
+        log.info(
+            f"Going to sleep for {sleep_time} Minute, for background cleanup to complete"
+        )
+        time.sleep(sleep_time * 60)
```

Comment:
why do we do `sleep_time * 60`? it could simply be `time.sleep(300)`

Reply:
@keesturam , you are right, but since i want the log to be in Minutes , so i used the variable `sleep_time` which is in minutes.